run-sandbox-runner.sh

```
@@ -3,16 +3,15 @@ set -euo pipefail
 
 cd /app/workspace 2>/dev/null || true
 
-if [[ -z "${MINDROOM_RUNTIME_PATHS_JSON:-}" ]]; then
-  export MINDROOM_RUNTIME_PATHS_JSON="$(
+if [[ -z "${MINDROOM_SANDBOX_STARTUP_MANIFEST_PATH:-}" ]]; then
+  export MINDROOM_SANDBOX_STARTUP_MANIFEST_PATH="$(
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)